src/data/amd_perf_events.rs

```diff
@@ -72,61 +73,61 @@ lazy_static! {
         },
         NamedCtr {
             name: "branch-mpki",
-            nrs: vec![BRANCHES],
+            nrs: vec![BRANCH_MISPRED],
             drs: vec![INSTRUCTIONS],
             scale: 1000
         },
         NamedCtr {
             name: "data-l1-mpki",
-            nrs: vec![L1_DATA],
+            nrs: vec![L1_DATA_FILL],
             drs: vec![INSTRUCTIONS],
             scale: 1000
         },
         NamedCtr {
             name: "inst-l1-mpki",
-            nrs: vec![L1_INSTRUCTIONS],
+            nrs: vec![L1_INSTRUCTION_MISS],
             drs: vec![INSTRUCTIONS],
             scale: 1000
         },
         NamedCtr {
             name: "l2-mpki",
-            nrs: vec![L2],
+            nrs: vec![L2_DEMAND_MISS],
             drs: vec![INSTRUCTIONS],
             scale: 1000
         },
         NamedCtr {
             name: "l3-mpki",
-            nrs: vec![L3],
+            nrs: vec![L1_ANY_FILLS_DRAM],
             drs: vec![INSTRUCTIONS],
             scale: 1000
```

Comment:
Name says L3, nrs says L1. Are you sure this is right?

Reply:
The counter is approximating L3 misses. "This is sorta l3 mpki, but ellides Prefetch misses from L2" - from our docs.